Update governance.md (#35)
frictionlessdata/datapackage: content/docs/standard/governance.md

  order: 7
---

> The Data Package Standard is governed by a working group made of [Frictionless](https://frictionlessdata.io/) community members coming from different disciplines. The composition of the working group is [publicly accessible on GitHub](https://github.com/orgs/frictionlessdata/teams/data-package-working-group/members?query=).

Reviewer: ```suggestion
The Data Package standard is governed by a working group made of [Frictionless](https://frictionlessdata.io/) community members coming from different disciplines. The composition of the working group is [publicly accessible on GitHub](https://github.com/orgs/frictionlessdata/teams/data-package-working-group/members).
```

I dropped the chaptalized S in standard, otherwise it looks like Data Package should be referred to as the Data Package Standard. Also removed the `query` parameter from the link (the page shows the same info).

Author: Thanks @peterdesmet!

> The working group works asynchronously, using a review model for any changes in the specifications. We also hold [monthly update calls](https://forms.gle/UEqGnYKESqqw1LeW7) for the working group, which are of course not compulsory, and which are also open to the broader community. 

Reviewer: ```suggestion
The working group works asynchronously, using a review model for any changes in the specification. We also hold [monthly update calls](https://forms.gle/UEqGnYKESqqw1LeW7) for the working group. Participation is optional and open to the broader community. 
```

Would refer to `specification` (singular, cf. documentation) as a whole.

## How we decide

> For every GitHub issue on the specs submitted by the community throughout the years, a working group member will propose a recommended action to the working group. The action proposed will be accepted if consensus with the working group is reached, meaning we have arrived at a decision, or at least a compromise, that everyone can live with.

Reviewer: ```suggestion
Changes to the specification can be suggested by anyone as a GitHub issue. For each issue, a working group member will propose a recommended action to the working group. The action proposed will be accepted if consensus with the working group is reached, meaning we have arrived at a decision, or at least a compromise, that everyone can live with.
```

The working group will be invited to share their view in a devoted GitHub issue. If a broader conversation is needed, the proposal discussion can be elevated to the monthly call for deliberation. The working group will be given a reasonable amount of time to review the proposed action.

Consensus is reached and the issue is closed if at least ⅔ of the working group members participate in the discussion and express their favourable opinion. In case of serious and explicitly stated concerns, working group members who are core library investors (at the moment: [Open Knowledge Foundation (OKFN)](https://okfn.org/), the [Research Institute for Nature and Forest (INBO)](https://www.vlaanderen.be/inbo/en-gb/homepage/), [Datopian](https://www.datopian.com/)) may veto a proposed action.

> The community manager at OKFN will reach out to working group members who did not participate in the discussion to make sure their opinion is also captured. We want to ensure that reminders of your participation are handled with care. You can expect a gentle and considerate approach, such as receiving an email once every two weeks highlighting any issues where your vote is pending. Our goal is to keep you informed without causing any unnecessary inconvenience, so you can feel confident and at ease with the process

Reviewer: This paragraph suddenly switches to the you and we form. Would adapt.
